frontend/src/components/CreateYourAccount/index.js

```diff
@@ -0,0 +1,203 @@
+    agreeToTerms: { value: false, isValid: false, isDirty: false },
+    isFormValid: false
+  });
+
+  const validation = (name, value) => {
+    switch (name) {
+      case "fullName":
+        return !!value && value.length < 100;
+      case "age":
+        return !!+value && +value > 0 && +value < 100;
+      case "gender":
+        return +value !== 0;
+      case "preexistingMedicalCondition":
+        return !value || value.length < 500;
+      case "quarantineStatus":
+        return !!value;
+      case "agreeToTerms":
+        return value;
+      default:
+        return false;
+    }
+  };
+
+  const submitForm = () => {
+    if (!form.isFormValid) {
+      return;
+    }
+
+    api.post(`http://localhost:5008/api/user`, {
+      fullName: form.fullName.value,
+      age: form.age.value,
+      gender: form.gender.value,
+      preexistingMedicalCondition: form.preexistingMedicalCondition.value,
+      quarantineStatus: form.quarantineStatus.value
+    });
+  };
+
+  const handleOnChange = (e, val) => {
+    const name = e.target.name;
+    const value = val !== undefined ? val : e.target.value;
+    const isValid = validation(name, value);
+
+    let isFormValid = isValid;
+    if (isValid) {
+      Object.keys(form).forEach(k => {
+        if (!isFormValid || k === "isFormValid") {
+          return;
+        } else if (k && k !== name) {
+          isFormValid = form[k].isValid;
+        }
+      });
+    }
+
+    setForm(form => ({
+      ...form,
+      [name]: {
+        value,
+        isValid,
+        isDirty: true
+      },
+      isFormValid
+    }));
+  };
+
+  const handleQuarantineStatusChange = e => {
+    setForm(form => ({
+      ...form,
+      quarantineStatus: {
+        value: e,
+        isValid: validation("quarantineStatus", e)
+      }
+    }));
+  };
+
+  const lifestyleOptions = [
+    { key: "2", value: "Auto izolare" },
+    { key: "3", value: "Carantină la domiciliu" },
+    { key: "4", value: "Carantină specializată" },
+    { key: "1", value: "Niciuna" }
+  ];
+
+  const genderOptions = [
+    { value: "0", text: "", selected: true },
+    { value: "1", text: "Feminin" },
+    { value: "2", text: "Masculin" }
+  ];
+
+  const genderDdlProps = {
+    onChange: function(el) {
+      // fix for missing name
+      el.target.name = "gender";
+      handleOnChange(el);
+    }
+  };
+
+  return (
+    <BasePage>
+      <SidebarLayout>
+        <Hero
+          title="Creează-ți cont"
+          subtitle="Având cont poți efectua formularele de evaluare zilnic și poți înregistra în contul tău membrii familiei care nu au e-mail sau acces la un calculator."
+          useFallbackIcon={true}
+        />
+        <form>
+          <Input
+            type="text"
+            name="fullName"
+            label="Nume și prenume"
+            onChange={handleOnChange}
+            color={
+              !form.fullName.isValid && form.fullName.isDirty ? "danger" : ""
+            }
+          />
```

Comment:
not sure if you forgot to add the email, password and confirm-password fields or do you plan to add them later, but i'll leave this comment just in case :D
![image](https://user-images.githubusercontent.com/6429818/78508220-bd679e00-778d-11ea-83f0-773e8a9ae313.png)